.github/workflows/functional.yml

```
@@ -86,8 +80,13 @@ jobs:
           SECRETS_RSTUF_ADMIN_PASSWORD: test-secret
 
     steps:
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
-      - uses: actions/setup-python@c4e89fac7e8767b327bbad6cb4d859eda999cf08
+      - name: Checkout release tag
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```

Nit: Maybe a small comment about which version of the action the hashes point to will be useful.
That's what I did in https://github.com/vmware/repository-service-tuf-api/pull/155.
As an example:
```suggestion
        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
```